### PR TITLE
Person service

### PR DESCRIPTION
**Pull request by Kutaiba-Nezar-Kashmar**
- new person service
- Component test
- endpoint

### CODE REVIEW

File: src/Services/Person/Person.API/Controllers/V1/PersonDetails/PersonDetailsController.cs [hunk 1]
```diff
@@ -0,0 +1,61 @@
+﻿using System.Net;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Person.API.Controllers.V1.PersonDetails.Dto;
+using Person.Application.FetchPersonDetails;
+
+namespace Person.API.Controllers.V1.PersonDetails;
+
+[ApiController]
+[Route("api/v1/person/details")]
```
Comment (michaelbui99) [inline src/Services/Person/Person.API/Controllers/V1/PersonDetails/PersonDetailsController.cs:10]: Add couch-potatoes to the route 🔥

File: src/Services/Person/Person.Test/FetchPersonDetails/ComponentTests/FetchPersonDetailsTest.cs [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        _loggerMock = new Mock<ILogger<FetchPersonDetailsRepository>>();
+        _personDetailsComparer = new PersonDetailsComparer();
+    }
+
+    [Test]
+    public async Task FetchPersonDetails_WithPersonID2_ShouldReturnCorrectData()
+    {
+        // Arrange
+        const string file =
+            "FetchPersonDetails/ComponentTests/Fakes/PersonDetailsWithId2Response.json";
```
Comment (michaelbui99) [inline src/Services/Person/Person.Test/FetchPersonDetails/ComponentTests/FetchPersonDetailsTest.cs:36]: Not sure how it works after compiling the project, but this path might break when running in CI environment 🔥

**Review by michaelbui99 — CHANGES_REQUESTED** [review]
🔥🔥🔥🔥

File: src/Services/Person/Person.API/Controllers/V1/PersonDetails/PersonDetailsController.cs (hunk 1 above)
Comment (Kutaiba-Nezar-Kashmar) [reply src/Services/Person/Person.API/Controllers/V1/PersonDetails/PersonDetailsController.cs:10]: Done 👍

File: src/Services/Person/Person.Test/FetchPersonDetails/ComponentTests/FetchPersonDetailsTest.cs (hunk 2 above)
Comment (Kutaiba-Nezar-Kashmar) [reply src/Services/Person/Person.Test/FetchPersonDetails/ComponentTests/FetchPersonDetailsTest.cs:36]: Yeah, it should read the relative path in the test project, so in theory, it should work 🤞

**Review by michaelbui99 — APPROVED** [review]
hot wok 🔥🔥🔥🔥

**Pull request merged**